cmds/exp/pxeserver/main.go

```diff
@@ -41,14 +41,15 @@ var (
 type server struct {
 	mac          net.HardwareAddr
 	yourIP       net.IP
+	subnet        net.IP
```

Comment:
is this formatted nicely?

Reply:
it is now. I forgot on osx I don't have my format-on-save stuff yet.

Reply:
fixed